config/import-control-assertions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<!DOCTYPE import-control PUBLIC
+    "-//Checkstyle//DTD ImportControl Configuration 1.4//EN"
+    "https://checkstyle.org/dtds/import_control_1_4.dtd">
+
+<import-control pkg=".*" regex="true">
+
+  <!-- Disallow obsolete Junit API -->
+  <disallow pkg="org.junit.jupiter.api.Assertions"/>
+  <allow pkg=".*" regex="true" />
```

Comment:
instead of it you can set `strategyOnMismatch=allowed` as property in `import-control`

Reply:
done